source/actions3/actions/AMAxisFinishedActionInfo.cpp

```diff
@@ -43,3 +43,8 @@ AMActionInfo3 *AMAxisFinishedActionInfo::createCopy() const
 	info->dissociateFromDb(true);
 	return info;
 }
+
+void AMAxisFinishedActionInfo::setAxisName(const QString &axisName)
+{
+	axisName_ = axisName;
```

Comment:
Ah yes. There will be a few more like this for added setters. I will go
through and add them all.